src/pages/Order/Order.scss

```
@@ -0,0 +1,170 @@
+@import '/src/styles/mixin.scss';
+
+.order {
+  @include flexCenter;
+  width: 100vw;
+  height: 100vh;
+  background-color: $Black;
+
+  .orderContainer {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: 1fr 3fr 12fr 4fr;
+    width: 1000px;
+    height: 800px;
+
+    .orderHeader {
+      @include flexCenter;
+      grid-column: span 4;
+      border-top-left-radius: 20px;
+      border-top-right-radius: 20px;
+      font-weight: 500;
+      font-size: 1.2rem;
+      background-color: $Brown;
+      color: white;
+    }
+
+    .productInfo {
+      grid-column: span 3;
+      background-color: $Gray33;
+      color: white;
+    }
+
+    .gradeInfo {
+    }
+
+    .concertHall {
+      display: grid;
+      grid-template-rows: 1fr 8fr;
+      grid-column: span 3;
+      background-color: $Gray33;
+
+      .stage {
+        @include flexCenter;
+      }
+      .stage div {
+        @include flexCenter;
+        width: 90%;
+        height: 80%;
+        margin-left: 2%;
+        border-bottom-left-radius: 50px;
+        border-bottom-right-radius: 50px;
+        font-weight: 700;
+        font-size: 1.4rem;
+        background-color: beige;
+      }
+
+      .seatTable {
+        display: grid;
+        grid-template-columns: 1fr 30fr;
+        grid-template-rows: 1fr 20fr;
+        gap: 2px;
+        padding: 2%;
+
+        .seatColumn {
+          display: grid;
+          grid-template-columns: repeat(30, 1fr);
+          color: white;
+
+          .column {
+            @include flexCenter;
+            font-size: 0.7rem;
+          }
+        }
+
+        .seatRow {
+          display: grid;
+          grid-template-rows: repeat(20, 1fr);
+          color: white;
+
+          .row {
+            @include flexCenter;
+            font-size: 0.7rem;
+          }
+        }
+
+        .seatList {
+          display: grid;
+          grid-template-columns: repeat(30, 1fr);
+          grid-template-rows: repeat(20, 1fr);
+          gap: 1px;
+
+          .seat {
+            width: 19px;
+            height: 19px;
+            border: 1px solid black;
+          }
+
+          .A {
+            background-color: rgb(242, 159, 6);
+            &:hover {
+              background-color: rgb(242, 159, 6, 0.7);
+            }
+          }
+          .B {
+            background-color: rgba(102, 51, 153);
+            &:hover {
+              background-color: rgba(102, 51, 153, 0.7);
+            }
+          }
+          .C {
+            background-color: #4682b4;
+            &:hover {
+              background-color: rgba(70, 130, 180, 0.7);
+            }
+          }
+          .D {
+            background-color: #999999;
+            &:hover {
+              background-color: rgba(153, 153, 153, 0.7);
+            }
+          }
```

seat class selector와 함께 쓰이는 부분들이기 때문에, seat class selector에 종속되도록 써주는 게 더 좋을 것 같습니다.

```suggestion
          .seat {
            border: 1px solid black;
            
            &.A {
              background-color: rgb(242, 159, 6);
              
              &:hover {
                background-color: rgb(242, 159, 6, 0.7);
              }
            }
            
            &.B {
              background-color: rgba(102, 51, 153);
             
              &:hover {
                background-color: rgba(102, 51, 153, 0.7);
              }
            }
            
            &.C {
              background-color: #4682b4;
            
              &:hover {
                background-color: rgba(70, 130, 180, 0.7);
              }
            }
            
            &.D {
              background-color: #999999;
              
              &:hover {
                background-color: rgba(153, 153, 153, 0.7);
              }
            }
          }
```